src/routes/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
this All routes file might end up being a gigantic ones you might want to try a different approach